### DISCUSSION TITLE
Autoupdate disabler not working

### DISCUSSION THREAD

**Discussion by jimmysofat6864** [SpotX-Official/SpotX#11] [Errors] [answered]
For some reason, my system still autoupdated even when I selected disable autoupdate. Does anybody else have this issue?

**Comment by amd64fox** [comment]
Hi, which version of Widows and Spotify are you using?  Were there any errors during the patch installation?

**Comment by jimmysofat6864** [comment, marked as answer]
Using latest version of Windows 10 and Spotify before it updated to a new
version. Patch installed successfully with no error messages.

**Reply by amd64fox** [reply]
Do it:
- Press Win + R
- Enter `%localappdata%\Spotify`
- Take a screenshot of the `Spotify` folder and send it here

**Reply by jimmysofat6864** [reply]
> Do it:
> 
> * Press Win + R
> * Enter `%localappdata%\Spotify`
> * Take a screenshot of the `Spotify` folder and send it here

![Screenshot 2021-09-13 162458](https://user-images.githubusercontent.com/69532417/133169849-c6332a82-39a5-4489-a511-89cc88eeec69.png)

**Reply by amd64fox** [reply]
- Run it again [Install.bat](https://github.com/amd64fox/SpotX/releases/download/1.0/Install.bat)
 - During the installation process, press `y` when you are asked about blocking updates.
 - After that, send again a screenshot of the same Spotify folder.

**Reply by jimmysofat6864** [reply]
> * Run it again [Install.bat](https://github.com/amd64fox/SpotX/releases/download/1.0/Install.bat)
> * During the installation process, press `y` when you are asked about blocking updates.
> * After that, send again a screenshot of the same Spotify folder.

![image](https://user-images.githubusercontent.com/69532417/133171899-95724efd-c821-4c25-b641-06d6d94070ec.png)

**Reply by amd64fox** [reply]
Now all updates should be blocked, this can be seen by the `Update` file that has now appeared in the Spotify folder.

Probably, you clicked the wrong button last time, so the script did not work, since no one has written me a similar error yet.

In general, let us know here when you have a new version, whether the blocker has worked or not.
